server/server.go

```diff
@@ -88,6 +88,9 @@ var (
 )
 
 const (
+	// RPCVersion should be bumped anytime changes are made which require
+	// the RPC client to upgrade to latest RPC server to be compatible
+	RPCVersion   string = "1.0.0"
```

Comment:
I think you need to set this in your proto file, don't you?

Reply:
Also, Avago uses an integer convention, not semantic version

Reply:
Constants are not supported in proto 3 AFAIK